ibc_data/conditions.csv

```diff
@@ -0,0 +1,377 @@
+Discount,delay,Effect of reward delay
+Discount,amount,Effect of reward gain
+SelectiveStopSignal,go_critical,Answer to the stim (critical side)
+SelectiveStopSignal,go_noncritical,Answer to the stim (noncritical side)
+SelectiveStopSignal,stop,Hold motor response
+SelectiveStopSignal,ignore,Answer regardless of the stop signal
+Stroop,congruent,Color and word are the same
+Stroop,incongruent,Color and word are different
+ColumbiaCards,num_loss_cards,Probability of losing
+ColumbiaCards,loss,Expected loss
+ColumbiaCards,gain,Expected gain
+DotPatterns,cue,
+DotPatterns,correct_cue_correct_probe,"Target pair, captures expectancy after correct cue"
+DotPatterns,correct_cue_incorrect_probe,Nontarget pair that also captures expectancy after correct cue
+DotPatterns,incorrect_cue_correct_probe,Incorrect pair. The probe is correct but the cue is not
+DotPatterns,incorrect_cue_incorrect_probe,"Incorrect pair, both are incorrect"
+WardAndAllport,planning_ambiguous_intermediate,Partially ambiguous goal state with intermediate movement
+WardAndAllport,planning_unambiguous_direct,Unambiguous goal state without intermediate movement
+WardAndAllport,planning_ambiguous_direct,Partially ambiguous goal state without intermediate movement
+WardAndAllport,planning_unambiguous_intermediate,Unambiguous goal state with intermediate movement
+BiologicalMotion1,global_upright,"Structural information is preserved, but individual local trajectories are mirror-symmetric. ""Global-only"" in the original paper."
+BiologicalMotion1,global_inverted,"Global walker, inverted upside-down"
+BiologicalMotion1,natural_upright,"Local information is preserved, but the points are randomly shuffled along the X-axis, rendering global cues uninformative. ""Local-natural"" in the original experiment."
+BiologicalMotion1,natural_inverted,"Local natural walker, inverted along the horizontal axis"
+BiologicalMotion2,modified_upright,"Neither structural or local information is carried out by this type of walker, it uses both types of modifications used for the previous two categories. ""Local modified"" in the original study."
+BiologicalMotion2,modified_inverted,"Local modified walker, inverted along the horizontal axis"
+BiologicalMotion2,natural_upright,"Local information is preserved, but the points are randomly shuffled along the X-axis, rendering global cues uninformative. ""Local-natural"" in the original experiment."
+BiologicalMotion2,natural_inverted,"Local natural walker, inverted along the horizontal axis"
+MathLanguage,arithmetic_fact_auditory,Listen to arithmetic fact
+MathLanguage,arithmetic_fact_visual,Read arithmetic fact
+MathLanguage,arithmetic_principle_auditory,Listen to arithmetic principle
+MathLanguage,arithmetic_principle_visual,Read arithmetic principle
+MathLanguage,colorlessg_auditory,Auditory Jabberwocky sentence parsing
+MathLanguage,colorlessg_visual,Visual Jabberwocky sentence parsing
+MathLanguage,context_auditory,Fixation

```

Comment:
Sounds wrong (?)

Reply:
It does haha, maybe @man-shu knows better?

Reply:
We modified the syntax to: Jabberwocky sentence presented as auditory/visual stimulus